tests/components/notify_events/test_notify.py

```diff
@@ -0,0 +1,38 @@
+"""The tests for notify_events."""
+from homeassistant.components.notify import ATTR_DATA, ATTR_MESSAGE, DOMAIN
+from homeassistant.components.notify_events.notify import (
+    ATTR_LEVEL,
+    ATTR_PRIORITY,
+    ATTR_TOKEN,
+)
+
+from tests.common import async_mock_service
+
+
+async def test_send_msg(hass):
+    """Test notify.events service."""
+    notify_calls = async_mock_service(hass, DOMAIN, "events")
+
+    await hass.services.async_call(
```

Comment:
This is just testing the mock service we registered above. We're not testing the integration.